pkgs/data/icons/volantes-cursors/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, inkscape
+, xcursorgen
+}:
+stdenv.mkDerivation {
+  pname = "volantes-cursors";
+  version = "2022-08-27";
+
+  src = fetchFromGitHub {
+    owner = "varlesh";
+    repo = "volantes-cursors";
+    rev = "b13a4bbf6bd1d7e85fadf7f2ecc44acc198f8d01";
+    hash = "sha256-vJe1S+YHrUBwJSwt2+InTu5ho2FOtz7FjDxu0BIA1Js=";
+  };
+
+  nativeBuildInputs = [ inkscape xcursorgen ];
+
+  buildPhase = ''
+    make build
```

Comment:
This can be achieved with `makeTargets`, which is nicer than overriding the buildPhase.

Reply:
Sounds good to me, just curious if you have a link to where it is defined in nixpkgs? Can't seem to find it or any docs on it.

Reply:
Ah its just passed as an env variable.